tools/update-tester/src/installer.rs

```diff
@@ -77,6 +90,36 @@ fn get_current_checkout() -> xshell::Result<String> {
     cmd!("git rev-parse --verify HEAD").read()
 }
 
+fn get_pgx_version(cargo_toml_contents: &str) -> String {
+    let mut version = get_field_val(cargo_toml_contents, "pgx").to_string();
+    version.truncate(3); // only care about the major.minor version
+    version
+}
+
+// find a `<field name> = "<field value>"` and extract `<field value>`
+fn get_field_val<'a>(contents: &'a str, field_name: &str) -> &'a str {
+    contents
+        .lines()
+        .filter(|line| line.starts_with(field_name))
+        .map(get_quoted_field)
+        .next()
+        .unwrap_or_else(|| panic!("cannot read field `{}` in file", field_name))
+}
+
+// given a `<field name> = "<field value>"` extract `<field value>`
+fn get_quoted_field(line: &str) -> &str {
```

Comment:
this is a near duplicate of [`tools/post-install/src/main.rs:get_quoted_field`](https://github.com/timescale/timescaledb-toolkit/blob/8183d126e4898d64892ee358b3b48b8851f7abbf/tools/post-install/src/main.rs#L274-L283)

Uses `"` instead of `'`, but if I'm going into the `post-install` crate for the other function, I might just change the function there to work for `"` too.

Reply:
Oh, wait, these aren't actually identical, the other one looks for things in  single quotes `'`, while this one looks for things in double quotes `"`.

Reply:
I have mixed feelings about this; on the one hand this is nice and simple and should work now. On the other hand the TOML syntax is much more involved than the `.control` one, and this fits us to one specific version. On the gripping hand, this version auto-discovery won't work for the fancier cargo features like git dependencies anyway, so whatever we choose we'll probably have to change if the code if we do that. 

In other words: you can use a real toml library if you want; this is always going to compile the current version of Toolkit, so the additional overhead shouldn't be that high, but don't feel like you have to. If you do decide to go that route, [`toml_edit`](https://docs.rs/toml_edit/latest/toml_edit/index.html) seems to be the one cargo is using right now, so that should be a safe choice.

Reply:
Sounds like we are on the same page. I was a little torn about hand-rolling my own `Cargo.toml` inspection but was also unsure about introducing a dependency for what seemed like a straightforward enough task (for now at least). 

`toml_edit` looks like good choice to me!